### ISSUE TITLE
Games Api: order games correctly; add # of results option; add powerful navigational option

### ISSUE THREAD

**Issue by ScalaWilliam** [ScalaWilliam/duel.gg#26]
Default # of limit = 10
Maximum # of limit = 50

Also add another lookup:
```
/center/:gameId/
```

This would place the matching gameId in the middle of the result-set. The game MUST exist though.
Why is this useful? Well, we can do things like:

```
/center/gameid/?radius=3
```
==> and then show at most 7 games in total.
```
{"previous": [gameid-1, gameid-2, ...], "focus": [gameid], "next": [gameid+1, gameid+2, ...]}
```

Then, we can do ```<link rel="next"```, ```<link rel="prev"``` and stuff to do context-based navigation. 

E.g. 'woop player duel games against rb player duel games' and such.

Very very powerful stuff. If radius = 1, then show only the game itself.
If no game in a radius, then exclude the field completely.

**Comment by ScalaWilliam** [comment]
This is actually probably more important than the 'before' and 'after' stuff, actually.

**Comment by ScalaWilliam** [comment]
Closed by #27
